Implemented getCodeText(string attr) in llvm/cuda codegen and added python bindings for it - #52974

huiguoo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#53664 Implemented getCodeText(string attr) in llvm/cuda codegen and added python bindings for it - #52974**

Differential Revision: [D26929204](https://our.internmc.facebook.com/intern/diff/D26929204)

huiguoo committed:
Implemented getCodeText(string attr) in llvm/cuda codegen and added python bindings for it - #52974

[ghstack-poisoned]

huiguoo committed:
Update on "Implemented getCodeText(string attr) in llvm/cuda codegen and added python bindings for it - #52974"


Differential Revision: [D26929204](https://our.internmc.facebook.com/intern/diff/D26929204)

[ghstack-poisoned]

huiguoo committed:
Update on "Implemented getCodeText(string attr) in llvm/cuda codegen and added python bindings for it - #52974"


Differential Revision: [D26929204](https://our.internmc.facebook.com/intern/diff/D26929204)

[ghstack-poisoned]

bertmaher reviewed (approved):
Looks good, just a few superficial comments.

ZolotukhinM commented:
Nit: if you add a "Fixes #XYZ" line to the body of your commit message, github will automatically close the #XYZ issue once the PR lands.

huiguoo committed:
Update on "Implemented getCodeText(string attr) in llvm/cuda codegen and added python bindings for it - #52974"


Differential Revision: [D26929204](https://our.internmc.facebook.com/intern/diff/D26929204)

[ghstack-poisoned]
